lib/processors/lib/utils.py

```diff
@@ -11,7 +12,11 @@ def get(self):
             userid = self.get_argument('userid')
             privatekey_pem = self.get_argument('privatekey', None)
             publickey_pem = self.get_argument('publickey', None)
-            user = User(userid, publickey_pem=publickey_pem, privatekey_pem=privatekey_pem)
+            user_blob = get_user(userid)
```

Comment:
Ahh I see the problem. The issue is when the scrapers run the first time around, they create a user object that needs a name for the processors to use. However, the API handler inherits the same User object but at that point it doesn't have a username which was creating errors for peyman last night.

So we just have to decide, do we want to make it such that the handlers instance of the User object just has the name set to None or something?